config/application.rb

```diff
@@ -66,6 +66,16 @@ class Application < Rails::Application
 
     # Caches user information in orders for the specified period of time
     config.orders_user_cache_expiration = 15.minutes
+    
+    # Make `form_with` generate non-remote forms.
+    config.action_view.form_with_generates_remote_forms = false
+    
+    # Store boolean values are in sqlite3 databases as 1 and 0 instead of 't' and
+    # 'f' after migrating old data.
+    config.active_record.sqlite3.represent_boolean_as_integer = true
```

Comment:
Is true the default value?

Reply:
yes, but there is a note in the code that indicates that it may be removed from the defaults list.  No explanation, so I thought it best to retain the current behavior.